flight/Modules/UAVOFrSKYSensorHubBridge/UAVOFrSKYSensorHubBridge.c

```diff
@@ -400,6 +405,10 @@ static void uavoFrSKYSensorHubBridgeTask(void *parameters)
 			// As long as there is no voltage for each cell
 			// all cells will have the same voltage.
 			// Receiver will know number of cells.
+			uint8_t cellCount = batState.DetectedCellCount;
+			if(cellCount)
+				batSettings.NbCells = cellCount;
+
```

Comment:
I tested this and really read this code now.

I think we should only do this if cellcount is 0, and let you set cellcount to 0 in the UI to indicate autodetection.